src/components/CourseInformation/CourseInformation.js

```diff
@@ -147,7 +148,7 @@ const CourseInformation = () => {
         <div className="">
           <div className="float-left space-y-2 pr-5 pb-1">
             <CourseImage
-              image={courseInfo.data?.Technical_Information?.Thumbnail}/>
+              image={courseInfo.data?.Technical_Information?.Thumbnail || configImage}/>
```

Comment:
This should be coming from the config in redux. Something like configuration.course_img_fallback should work